### ISSUE TITLE
[FEATURE REQUEST] Add taskbar control buttons

### ISSUE THREAD

**Issue by TheJoin95** [nukeop/nuclear#542]
**Add control buttons on hover event of the icon app**

Let's say we want to change or stop the current track without opening the music player.
I suggest you to put these shortcut buttons like spotify or windows media player does:
![image](https://user-images.githubusercontent.com/6616203/67193257-4dd73f80-f3f5-11e9-8016-47db8e8f01a4.png)

Here are the docs for [Windows](https://electronjs.org/docs/tutorial/windows-taskbar) and [MacOS](https://electronjs.org/docs/tutorial/macos-dock).

These shortcut will be useful and it may help the user to see also which track is playing by the cover  image.

**Comment by nukeop** [comment]
This is a nice idea, and I might be able to test it on a VM. The Mac version of this feature will have to be handled by contributors. I'd also like to add these controls to the tray menu, and allow minimizing the program to tray.

**Comment by nukeop** [comment]
This could also include desktop actions: https://electronjs.org/docs/tutorial/linux-desktop-actions

**Comment by TheJoin95** [comment]
Yes, of course it could be implemented also the desktop actions for linux.
I can try to implement and test the Mac version of this, but I'm a newbie in Electron and I don't know if I'm able to accomlished this. 

Any help is welcome, thank you

**Comment by nukeop** [comment]
Nice, if you have any questions, let me know.

**Comment by charjac** [comment]
I will try to add this in my next PR. (really not a big deal).
@nukeop If you have png/svg for the play/pause/next/previous button, i take them.
@nukeop I can also include support for macos touchbar if you want. I checked the doc and it is really not a big deal.
This PR will also include mpris equivalent for Mac and windows.

**Comment by nukeop** [comment]
That would be really great. I have access to a Mac now so I can do that too, in case it gets boring for you.

For icons, since we already have Font Awesome, we could download svgs of corresponding icons and use that (can't copy direct links because they're generated but there's a "Download SVG" button under the header):

https://fontawesome.com/icons/play?style=solid
https://fontawesome.com/icons/pause?style=solid
https://fontawesome.com/icons/step-forward?style=solid
https://fontawesome.com/icons/step-backward?style=solid
https://fontawesome.com/icons/forward?style=solid
https://fontawesome.com/icons/backward?style=solid

**Comment by charjac** [comment]
Great !!! thanks !!!
it is really nothing. I already played with it a lot and the api is dead simple, 10 min max and its shipped ;).
